packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -0,0 +1,153 @@
+:local {
+  .checkbox {
+    color: var(--terra-form-checkbox-font-color, rgb(0, 0, 0));
+    display: block;
+    font-family: inherit;
+    margin-left: var(--terra-form-checkbox-container-margin-left, none);
+    margin-right: var(--terra-form-checkbox-container-margin-right, 2rem);
+    margin-top: var(--terra-form-checkbox-container-margin-top, 0.625rem);
+
+    /* Hitbox and container focus style will only surround checkbox + label, instead of the div block */
+    /* Padding puts space for container focus style */
+    .inner-div {
+      display: inline-block;
+      padding-left: var(--terra-form-checkbox-inner-div-padding-left, 0.214rem);
+      padding-right: var(--terra-form-checkboc-innter-div-padding-right, 0.214rem);
+    }
+
+    /* Hide label if isLabelHidden true */
+    .label-hidden {
```

Comment:
Instead of using css to hide the text we could just not render the text at all and set it as the aria-label attribute on the input.

Reply:
Possibly, I lean more towards `<label>some text visually hidden with css</label>` than `<label aria-label="some text in aria label"></label>`. While both are similar, I think the first will have wider support for communicating the label text in various applications (screen readers, etc).